src/main/java/net/pms/util/Constants.java

```diff
@@ -11,16 +11,19 @@
 	public static final String CHARSET_ISO_8859_7	= "ISO-8859-7".intern();
 	public static final String CHARSET_ISO_8859_8	= "ISO-8859-8".intern();
 	public static final String CHARSET_ISO_8859_9	= "ISO-8859-9".intern();
+	public static final String CHARSET_ISO_8859_11	= "ISO-8859-11".intern();
```

Comment:
@Sami32 those constants are used for automatic charset detection by ICU4J. The Thai language is unfortunatelly not supported so adding it here does nothing. See http://userguide.icu-project.org/conversion/detection#TOC-Detected-Encodings